source/core/NativeMemory.cpp

```diff
@@ -303,6 +303,9 @@ namespace GTA
 			address = FindPattern("\xB2\x01\xE8\x00\x00\x00\x00\x33\xC9\x48\x85\xC0\x74\x3B", "xxx????xxxxxxx");
 			_playerAddressFunc = reinterpret_cast<uintptr_t(*)(int)>(*reinterpret_cast<int *>(address + 3) + address + 7);
 
+			address = FindPattern("\x74\x21\x48\x8B\x48\x20\x48\x85\xC9\x74\x18\x48\x8B\xD6\xE8", "xxxxxxxxxxxxxxx") - 10;
+			_ptfxAddressFunc = reinterpret_cast<uintptr_t(*)(int)>(*reinterpret_cast<int*>(address) + address + 4);
```

Comment:
Why is the logic like this?

`(*(int*)address) + address + 4`

So if `address` is 0x100000 which holds a pointer to 0x170000, the result would be 0x270004...this doesn't make sense to me.